onnx/checker.py

```diff
@@ -90,6 +95,10 @@ def check_model(model, full_check=False):  # type: (Union[ModelProto, Text], boo
         C.check_model_path(model)
         m = onnx.load(model)
     else:
+        # If the protobuf is larger than 2GB,
+        # remind user should use the model path to check
+        if sys.getsizeof(model.SerializeToString()) > MAXIMUM_PROTOBUF:
```

Comment:
Better to avoid serializing the model twice.

Reply:
Good catch.